assembly/std/address.ts

```diff
@@ -61,20 +75,25 @@ export class Address implements Valider, Serializable {
   /**
    * Tests if two addresses are identical.
    *
-   * @param other - the address object to compare
+   * @param address - The address object to compare.
+   *
+   * @returns `true` if the addresses are identical, `false` otherwise.
    */
+
   @operator('==')
-  equals(other: Address): boolean {
-    return this._value == other.toString();
+  equals(address: Address): boolean {
+    return this._value == address.toString();
   }
 
   /**
    * Tests if two addresses are different.
    *
-   * @param other - the address object to compare
+   * @param address - The address object to compare.
```

Comment:
```suggestion
   * @remarks
   * This method compares the string representation of the current `Address` object to another `Address` object to determine if they are identical.
   *
   * @param address - The address object to compare.
```

Reply:
This one compare 2 address object, not really the string représentation. We can change the code to compare the string representation by calling toString. Also it's to check if they are different not identical.

Reply:
@Ben-Rey We are indeed comparin gthe string representation of the address because we are using the Address.equals function define just above

Reply:
Ok sorry I wasn't aware of the   @operator('==')